### DISCUSSION TITLE
I dont get it - How the server can be run on Rpi ?

### DISCUSSION THREAD

**Discussion by Behinder** [Barracuda09/SATPI#154] [General]
Server must get input from DVBS, how do you connect satellite cable to Rpi?
You don’t mention that all in the docs and if you need another box than using Rpi is redundant

**Comment by Barracuda09** [comment]
Hi @Behinder 

You can not connect it 'directly'  to the RPi, you need to use a USB tuner something like:
https://shop.sundtek.com/en/DVB-S-S2-S2X/Sundtek-SkyTV-Dual-2x-DVB-S-S2-S2X.html

**Comment by Behinder** [comment]
Thanks for explanation. seems to be intresting setup to build :)
